test/pages/IsSaleBetweenRelatedPartiesPageSpec.scala

```diff
@@ -28,4 +31,18 @@ class IsSaleBetweenRelatedPartiesPageSpec extends PageBehaviours {
 
     beRemovable[Boolean](IsSaleBetweenRelatedPartiesPage)
   }
+  "cleanup" - {
+    "should reset ExplainHowPartiesAreRelatedPage" - {
+      "when IsSaleBetweenRelatedPartiesPage is changed to No" in {
+        val emptyUserAnswers = UserAnswers("id", DraftId(1))
+
+        val ua = emptyUserAnswers
+          .set(ExplainHowPartiesAreRelatedPage, "reasons")
+          .get
+
+        IsSaleBetweenRelatedPartiesPage.cleanup(Some(false), ua) mustBe
+          Success(emptyUserAnswers)
+      }
+    }
```

Comment:
We want to be testing all scenarios here, namely what happens when the cleanup is called with a None, and Some(true).